src/snippets/muiPaper.ts

```diff
@@ -0,0 +1,8 @@
+import snip from '../snip'
+
+export const description = 'Material-UI <Paper>'
+export const body = snip`
+<Paper>
```

Comment:
I'll plan to also add a `muiPaperFull` snippet that allows you to select the `elevation` and `variant`